src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -112,6 +112,10 @@ def get(self, request, organization):
             "organizations:dynamic-issue-counts", organization, actor=request.user
         )
 
+        include_inbox = features.has(
+            "organizations:inbox", organization, actor=request.user
+        ) and "group_inbox" in request.GET.getlist("expand", [])
```

Comment:
```suggestion
        ) and "inbox" in request.GET.getlist("expand", [])
```

Using `inbox` makes the request parameter and the attribute in the response match.